src/app/directives/resize-column.directive.ts

```diff
@@ -0,0 +1,144 @@
+import {ElementRef, Renderer2, HostListener, Directive, AfterViewInit, Input} from '@angular/core';
+import {SettingsService, SettingsProperties} from '../services/settings.service';
+import {Store} from 'src/app/stores/store';
+
+export type ColumnType = 'routeMenu' | 'envLogs';
+
+/**
+ * Allows the resizing of the parent element using css(width,)
+ * On mousedown registers listeners into the body (mousemove,mouseup) to detect resizing.
+ *
+ * Min width is passed as a parameter and max is calculated based on body size and a factor.
+ */
+
+@Directive({
+  // tslint:disable-next-line:directive-selector
+  selector: 'resize-column'
+})
+export class ResizeColumnDirective implements AfterViewInit {
+
+  constructor(private elementRef: ElementRef,
+              private settingsService: SettingsService,
+              private store: Store,
+              private renderer: Renderer2) {
+    elementRef.nativeElement.style.cursor = 'col-resize';
```

Comment:
This should be moved to the styles.scss file https://github.com/mockoon/mockoon/pull/264/files#diff-354873b4b545926f38dc09efd5af2b6dR315